installation_policies/10.md

```diff
@@ -1,7 +1,7 @@
 10. Determine other package libraries and include directories.
 
-    1. `--with-<package>  --with-<package>-lib="linkable list of libraries” --with-package-include=”-I list of include 
```

Comment:
I'm not sure if we should be changing this autoconf part - i.e retain the --with-package-include and --with-pacakge-lib options?

Reply:
I am not too familiar with autoconf and this policy has not been an issue with any packages that use autoconf, so we probably should retain it. I will add it back in unless anyone else has objections.

Reply:
The vote today included retaining this line in.